API/controller/mealController.js

```diff
@@ -0,0 +1,114 @@
+import mealData from "../utils/mealData";
+import mealModel from "../model/mealModels";
+
+const MealController = {
+  fetchAllMeals(req, res) {
+    return res.status(200).json({
+      status: 200,
+      data: mealData
+    });
+  },
+  addAMeal(req, res) {
+    const { name, size, price, summary, imageUrl } = req.body;
+
+    const newlyCreatedMeal = {
+      id: mealData[mealData.length - 1].id + 1,
+      name,
+      size,
+      price,
+      summary,
+      imageUrl
+    };
+
+    const data = mealData.push(newlyCreatedMeal);
+    if (data) {
```

Comment:
Can you explain the rationale behind this? Why are you checking for the returned length of the new array?